### PR TITLE
added {transform} option.

### PR DESCRIPTION
**Pull request by stephenlb**
Transformation Option!

> return top 10 items sorted.

``` javascript
eon.chart({
    channel   : 'ermahgerd',
    transform : function(m) {
        return [{ latlng: [m.lat, m.lng] }];
    }
});
```

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -35,7 +35,7 @@ Parameter | Value | Default
 | id | The ID of the element where the map will be rendered. | ```undefined```
 | mb_token | Mapbox API Token. | ```undefined```
 | mb_id | Mapbox Map ID. | ```undefined```
-| subscribe_key | Your PubNub subscribe_key | ```demo```
```
Comment (ianjennings) [inline README.md]: whyyyyyyyyyyyy

Comment (stephenlb) [inline README.md:38]: because it ain't used!

File: pubnub-mapbox.js [hunk 2]
```diff
@@ -224,4 +226,4 @@ eon.m = {
 };
 eon.map = function(o) {
   return new eon.m.create(o);
-};
```
Comment (stephenlb) [inline pubnub-mapbox.js:227]: this is annoying I donno why it thinks this is a change....

Comment (ianjennings) [inline pubnub-mapbox.js:227]: it's the newline char

File: README.md (hunk 1 above)
Comment (ianjennings) [inline README.md:38]: yeah makes sense

**Pull request merged**